docker-compose.yml

```diff
@@ -1,5 +1,14 @@
 version: "3"
 services:
+  auth-db:
+    image: mysql
+    command: --default-authentication-plugin=mysql_native_password
+    restart: always
+    env_file:
+      - .env
+    network_mode: "host"
+    ports:
+      - '3306:3306'
```

Comment:
Is this port open to services outside docker ? Since this wont be necesarry in my opinion.

Reply:
Yes. It is. Right now we had to do this since. There is network bridge. So containers won't be able to talk to each other without it